articles/quickstart/spa/_includes/_authz_protect_client_side_routes.md

```diff
@@ -0,0 +1,3 @@
+## Protect Client-Side Routes
+
+You may want to give access to some routes in your application only to authenticated users. You can check if the user is authenticated with the ${check}.
```

Comment:
This will cause a build error, you need to escape `${check}` https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#escaping-strings